src/test/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceServiceTest.java

```diff
@@ -134,98 +146,109 @@ public void getObjects() throws Exception {
     }
 
     @Test
-    public void findByQueryParametersStringParameters() throws Exception
-    {
+    public void findByQueryParameters_ObjectUrnLike() throws Exception {
```

Comment:
I broke the individual groupings of tests into separate methods.  This way provides a more granular look at what is working and what is broken. This way if objectLike gets broken the other tests will tell us if they are still good or not.  When all the tests are in one method you have to work through broken code in a `failing further` approach where each fix exposes the next breakage.

We could break each test into an individual method for the best granularity but I don't think that is warranted in this situation.